platform/darwin/test/MGLOfflineStorageTests.mm

```diff
@@ -342,7 +340,7 @@ - (void)testAddFileContent {
         [self waitForExpectationsWithTimeout:5 handler:nil];
```

Comment:
Would you mind also bumping this timeout to 10 seconds? Xcode 10 on CircleCI is, uhh... a bit unreliable for these.